create.sql

```diff
@@ -1261,7 +1302,7 @@ ALTER TABLE ONLY step_science
 REVOKE ALL ON SCHEMA public FROM PUBLIC;
 REVOKE ALL ON SCHEMA public FROM postgres;
 GRANT ALL ON SCHEMA public TO postgres;
-GRANT ALL ON SCHEMA public TO rnorris;
+GRANT ALL ON SCHEMA public TO swalker;
```

Comment:
Is there a way to solve this issue so it doesn't change every time someone commits an update to the schema?